src/ontology/components/catbreeds.owl

```diff
@@ -14812,6 +14812,127 @@
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/VBO_0300019"/>
+                <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/VBO_0100259"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <obo:IAO_0000233 rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">https://github.com/monarch-initiative/vertebrate-breed-ontology/issues/161</obo:IAO_0000233>
+        <terms:contributor rdf:resource="https://orcid.org/0000-0002-4142-7153"/>
+        <terms:contributor rdf:resource="https://orcid.org/0000-0002-5002-8648"/>
+        <terms:contributor rdf:resource="https://orcid.org/0000-0002-5520-6597"/>
+        <terms:source>PMID:38644700</terms:source>
+        <terms:source rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">http://messybeast.com/blue-eye-breeds.htm</terms:source>
+        <oboInOwl:hasExactSynonym>Topaz</oboInOwl:hasExactSynonym>
+        <rdfs:label>Topaz (Cat)</rdfs:label>
+    </owl:Class>
+    <owl:Axiom>
+        <owl:annotatedSource rdf:resource="http://purl.obolibrary.org/obo/VBO_0100260"/>
+        <owl:annotatedProperty rdf:resource="http://www.w3.org/2000/01/rdf-schema#subClassOf"/>
+        <owl:annotatedTarget rdf:resource="http://purl.obolibrary.org/obo/VBO_0400000"/>
+        <oboInOwl:source>https://orcid.org/0000-0002-4142-7153</oboInOwl:source>
+        <oboInOwl:source>https://orcid.org/0000-0002-5002-8648</oboInOwl:source>
+    </owl:Axiom>
+    <owl:Axiom>
+        <owl:annotatedSource rdf:resource="http://purl.obolibrary.org/obo/VBO_0100260"/>
+        <owl:annotatedProperty rdf:resource="http://www.geneontology.org/formats/oboInOwl#hasExactSynonym"/>
+        <owl:annotatedTarget>Topaz</owl:annotatedTarget>
+        <oboInOwl:hasSynonymType rdf:resource="http://purl.obolibrary.org/obo/vbo#most_common_name"/>
+        <oboInOwl:source>PMID:38644700</oboInOwl:source>
+        <oboInOwl:source rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">http://messybeast.com/blue-eye-breeds.htm</oboInOwl:source>
+    </owl:Axiom>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/VBO_0100261 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/VBO_0100261">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/NCBITaxon_9685"/>
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/VBO_0400000"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/VBO_0300019"/>
+                <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/VBO_0100051"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/VBO_0300019"/>
+                <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/VBO_0100052"/>
```

Comment:
'has foundation stock" missing a source

Reply:
The source is a PMID. I put it in a column labelled ">A oboInOwl:source SPLIT=|" I am not sure why it is not showing?

Reply:
The source is a PMID. I put it in a column labelled ">A oboInOwl:source SPLIT=|" I am not sure why it is not showing?

Reply:
done